build.gradle

```diff
@@ -6,7 +6,7 @@ plugins {
 }
 
 ext.githubUri = 'https://github.com/vlingo/'
-ext.checkoutDir = '../'
+ext.checkoutDir = '/Users/wwerner/Projects/vlingo/'
```

Comment:
This will break all other developer builds. It is intended that the directory structure be:

```
vlingo/
  vlingo-actors/
  vlingo-auth/
  ...
```

Reply:
Yes, this was just a preliminary test I commited accidentally.
The initial state w/ `../` didn't work for me, so I tried to get it to build first. Anyway, it's gone now as you noticed